build.gradle

```diff
@@ -1,11 +1,12 @@
-buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:0.6'
-    }
+plugins {
+    id 'groovy'
+    id 'idea'
+    id 'maven-publish'
+    id 'java-gradle-plugin'
+    id 'com.jfrog.bintray'              version '1.8.4'
+    id 'net.nemerosa.versioning'        version '2.7.1'
+    id 'com.github.ben-manes.versions'  version '0.20.0'
+    id 'com.gradle.plugin-publish'      version '0.10.0'
```

Comment:
For this plugin we won't need the portal publish plugin. It's already synced from JCenter to the portal. Can you remove this plugin and its configuration below?

Reply:
Hmm OK, though I think this was the preferred way to publish plugins

Reply:
It is the preferred way but it's already set up to sync automatically. Not sure how the portal would behave if you'd actually publish there and sync.